src/tools/get-messages.ts

```diff
@@ -68,7 +68,7 @@ export const getMessages = memoizeOne((hacs: Hacs): Message[] => {
   if (repositoriesRestartPending.length > 0) {
     messages.push({
       name: hacs.localize("entry.messages.restart.title"),
-      path: "/config/server_control",
+      path: "/config/system",
```

Comment:
This will break all < 2022.5
Use `isComponentLoaded(this.hass, "my") ? "/_my_redirect/server_controls" : undefined` instead

Reply:
I have no idea what this means or does. Please commit this yourself. :)